optimize/upgrade/src/main/java/io/camunda/optimize/upgrade/main/UpgradeMain.java

```diff
@@ -44,30 +44,29 @@ public class UpgradeMain {
   }
 
   public static void main(final String... args) {
-
-    final boolean initSchemaEnabled =
-        ConfigurationServiceBuilder.createDefaultConfiguration()
-            .getElasticSearchConfiguration()
-            .getConnection()
-            .isInitSchemaEnabled();
-
-    final boolean clusterTaskCheckingEnabled =
-        ConfigurationServiceBuilder.createDefaultConfiguration()
-            .getElasticSearchConfiguration()
-            .getConnection()
-            .isClusterTaskCheckingEnabled();
-    if (!initSchemaEnabled || !clusterTaskCheckingEnabled) {
-      throw new UpgradeRuntimeException(
-          "Can't upgrade without enable cluster task checking " + "and init schema privileges");
-    }
-
     try {
       final DatabaseType databaseType =
           ConfigurationService.convertToDatabaseProperty(
               Optional.ofNullable(System.getenv(CAMUNDA_OPTIMIZE_DATABASE))
                   .orElse(ELASTICSEARCH_DATABASE_PROPERTY));
       LOG.info("Identified {} Database configuration", databaseType.getId());
 
+      final boolean initSchemaEnabled =
+          ConfigurationServiceBuilder.createDefaultConfiguration()
+              .getElasticSearchConfiguration()
+              .getConnection()
+              .isInitSchemaEnabled();
+
+      final boolean clusterTaskCheckingEnabled =
+          ConfigurationServiceBuilder.createDefaultConfiguration()
+              .getElasticSearchConfiguration()
+              .getConnection()
+              .isClusterTaskCheckingEnabled();
+      if (!initSchemaEnabled || !clusterTaskCheckingEnabled) {
```

Comment:
```suggestion
      if (databaseType == DatabaseType.ELASTICSEARCH && (!initSchemaEnabled || !clusterTaskCheckingEnabled)) {
```
This is more explicitly what I meant above, that this check is only important for ES

Reply:
fixed